detox/e2e/support/ui/screen/manage_channel_members.ts

```
@@ -132,11 +132,16 @@ class ManageChannelMembersScreen {
         }
-        await userDisplayName.tap();
+
+        // Corner-tap: the member row's center is obscured by the manage-members
+        // modal's UITransitionView (same workaround as PostOptionsScreen.deletePost).
+        await userDisplayName.tap({x: 1, y: 1});
         await wait(timeouts.TWO_SEC);
 
-        await expect(this.removeButton).toBeVisible();
-        await this.removeButton.tap();
+        await waitFor(this.removeButton).toBeVisible().withTimeout(timeouts.TEN_SEC);
+        // Corner-tap: remove row action is obscured by the manage-members modal's
```

ESLint

[error] Expected line before comment.

(lines-around-comment)